src/odemis/acq/fastem.py

```diff
@@ -567,15 +570,17 @@ def run(self):
             self.pre_calibrate(self._pre_calibrations)
             self._roa.overlap = overlap_init  # set back the overlap to the initial value
 
+        # set the sub-directories (<user>/<project-name>/<roa-name>)
+        # FIXME use username from GUI when that is implemented
+        username = self._detector.getMetadata().get(model.MD_USER, "fastem-user")
```

Comment:
I don't see where it's defined in the driver. For now, nowhere? Or only in the microscope file? Is it just used to define the root directory of the filename? If so, please update the fastem microscope files examples. If it's only used in the GUI, why do you need to use a metadata on the detector?
The externalStorage argument already comes with a username key, used to define the FTP credentials. What's the difference?

Reply:
The username of the externalStorage is the FTP username. The users want to be able to define a username for a certain project on the system, so that it gets stored as `/asm_service/[user]/[Project-ID]/[ROA-ID]/[Zpos][DATE-TIME]/*.tiff`. A single user can then find back all their projects in `/asm_service/[user]/`. 

I will update the fastem microscope files to include this